Resources/doc/index.rst

```diff
@@ -1,334 +1,42 @@
-Provides integration for DoctrineExtensions_ for your Symfony2 Project.
+StofDoctrineExtensions
```

Comment:
the title is wrong. There is no StofDoctrineExtensions, only a StofDoctrineExtensionsBundle